cpp/cmake/Dependencies.cmake

```diff
@@ -38,15 +38,16 @@ else(DEFINED ENV{RAFT_PATH})
   set(RAFT_DIR ${CMAKE_CURRENT_BINARY_DIR}/raft CACHE STRING "Path to RAFT repo")
 
   ExternalProject_Add(raft
-    GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           b58f97f2b5382a633e43daec31b26adf52e19a3b
+    GIT_REPOSITORY    https://github.com/aschaffer/raft.git
```

Comment:
I'm guessing before this PR is merged, this will be updated to rapidsai's repo and the git commit will also be updated accordingly?

Reply:
Yes. Comming soon, as the raft PR has been merged.